canister/src/api/send_transaction.rs

```diff
@@ -34,7 +34,12 @@ pub async fn send_transaction(request: SendTransactionRequest) {
         },
     )
     .await
-    .expect("Sending transaction bitcoin network must succeed");
+    .map_err(|(code, msg)| {
+        format!(
+            "Sending transaction bitcoin network must succeed, Rejection code: {:?}, {}",
+            code, msg
+        )
+    })
```

Comment:
This is not an error based on user input, so the correct behavior here would be to trap and not reject, so I'd revert the changes here.

Reply:
Good point, I agree.